consensus/clique/api.go

```diff
@@ -196,7 +196,11 @@ func (sb *blockNumberOrHashOrRLP) UnmarshalJSON(data []byte) error {
 	if err := json.Unmarshal(data, &input); err != nil {
 		return err
 	}
-	sb.RLP = hexutil.MustDecode(input)
+	unhexed, err := hexutil.Decode(input)
```

Comment:
Please call it `blob` :) We use it throughout the code when referring to a decoded binary... blob :)

Reply:
Fixed. Missed opportunity to call it `bob` :)